### PR TITLE
Second Pull request for Portfolio_site (Work about myself branch)

### PR DESCRIPTION
**Pull request by zuhebahmed88091**
Previously added  nav and headline contents. In this part added ->

- The middle part of the project which is work section (Projects)
- About myself part
- Language frameworks and skills parts

For this used the following tools ->

- CSS Flexbox
- CSS Grid
- HTML Semantics
- HTML Box Model

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -70,5 +70,218 @@ <h2>LET'S CONNECT</h2>
...
+            no accounts or sign-ups required.
+         </p>
+         <ul class="tools">
+          <li class="tools-items">HTML</li>
+          <li class="tools-items">CSS</li>
+          <li class="tools-items">JAVASCRIPT</li>
+         </ul>
+         <button class="user-buttons">
+          See Project
+         </button>
```
Comment (roshan-bajgain) [inline index.html]: - [x] According to the project requirement you should stick to the design as much as possible like color, font and spacing. I would suggest you please use the same background for buttons ie: `background: #FFFFFF;`
- [x] Please implement to all buttons. Also please increase the font weight for heading Tonic and please decrese the size to it looks like figma design. PLease implement for all section.
### Your design
his

### Figma design
fimga

File: index.html [hunk 2]
```diff
@@ -70,5 +70,218 @@ <h2>LET'S CONNECT</h2>
...
+                <p>JavaScript</p>
+              </section>
+            </article>
+
+            <article class="language-name">
+              <section class="language-logo">
+                <img src="./Properties/css.png" alt="">
+              </section>
+              <section class="certain-language-name">
+                <p>JavaScript</p>
```
Comment (roshan-bajgain) [inline index.html]: - [x] I would suggest you please list each category(e.g., "JavasScript", "HTML" and "CSS") in ul and each of them a li for better semantic HTML.

File: index.html [hunk 3]
```diff
@@ -70,5 +70,218 @@ <h2>LET'S CONNECT</h2>
...
+            </article>
+          </li>
+          <li>
+            <article class="skill-title">
+              <h2>Skills</h2>
+              <img src="./Properties/right-arrow.svg" alt="">
+            </article>
+          </li>
+        </ul>
+      </section>
```
Comment (roshan-bajgain) [inline index.html]: - [x] You should stick to the design as much as possible like color, font, spacing etc. Please use same color for `javascript, javascript, javascript,`.
- [x] [Optional] Also make `javascript, javascript, javascript, and change these 2 into HTML and CSS.
- [x] I would suggest you please make the list language, framework and skills that looks like figma deisgn. Please decrease the size.
- [x] Down arrow also looks different please make similar to the design.
### Your design
his

### Figma design
fimga

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @zuhebahmed88091, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- The design looks good ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -80,4 +87,258 @@
...
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .project-details-heading h2 {
+    font-weight: 700;
+    font-size: 2rem;
+    color: #172b4d;
+  }
```
Comment (franclobo) [inline style.css:132]: - **[OPTIONAL]** Well done loading the font family Poppins, but using the corresponding font styles could be better. Please add the font family to the card headers to be similar to the Figma design 👇🏽 

image

File: style.css [hunk 5]
```diff
@@ -80,4 +87,258 @@
...
+  .about-headline {
+    padding-top: 17%;
+    padding-left: 6%;
+    padding-right: 6%;
+    padding-bottom: 17%;
+    margin-bottom: 3%;
+    display: flex;
+    flex-direction: column;
+    background-color: #fff;
+    border-top-right-radius: 15%;
```
Comment (franclobo) [inline style.css:221]: - **[OPTIONAL]** It is better to use px units instead of percentages, I recommend using 100px to be similar to the Figma design.

**Review by franclobo — APPROVED** [review]
Hi @zuhebahmed88091, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 
- Good HTML and CSS practices ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 5 above)
Comment (zuhebahmed88091) [reply style.css:221]: But the instructions were to do not use px as units.

**Pull request merged**